tgui/packages/tgui/interfaces/MarionetteRemote.js

```diff
@@ -0,0 +1,163 @@
+import { useBackend } from '../backend';
```

Comment:
The interface should probably in typescript. See intrfaces in their individual folders with `index.tsx` and `types` for a start. The Apc one goes pretty deep.

`selected_command` and `status` would get to be enums.

Reply:
Once that is done, if you want to go for top notch organization, I would suggest moving the Controls and Implants sections to respective `Controls.tsx` and `Implants.tsx` files.

Reply:
Once that is done, if you want to go for top notch organization, I would suggest moving the Controls and Implants sections to respective `Controls.tsx` and `Implants.tsx` files.

Reply:
JS interfaces are fine, don't feel obligated to translate it to typescript.

Reply:
thank you for that. while I do want to learn typescript, I think I'll stick with the js one for now just because it's what I'm familiar with and it might take me a while to learn how to make a typescript port work. a tsx version could always be made later, so it's hardly a permanent loss